File: src/ios/FloatingWebview.m
#import <Cordova/CDVPlugin.h> // this already includes Foundation.h

@interface SignInWithApple : CDVPlugin {
  NSMutableString *_callbackId;
}
@end

@implementation FloatingWebview
- (void)pluginInitialize {
  NSLog(@"FloatingWebview initialize");
}

- (void)show:(CDVInvokedUrlCommand *)command {
  NSLog(@"show!");
}

- (void)show:(CDVInvokedUrlCommand *)command {
  NSLog(@"hide!");
}

File: www/floating-webview.js
var exec = require('cordova/exec');

exports.show = function(arg0, success, error) {
  exec(success, error, "FloatingWebview", "show", [arg0]);
};

exports.hide = function(arg0, success, error) {
  exec(success, error, "FloatingWebview", "hide", [arg0]);
};
